### PR TITLE
Feature/rac string lines sequence

### PR DESCRIPTION
**Pull request by atmadev**
Ticket:
https://techery.atlassian.net/browse/DT-2095

These changes are needed for pull request:
https://github.com/techery/DreamTrips-iOS/pull/1956

Added RAC line sequence

### CODE REVIEW

File: ReactiveCocoa.podspec.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "ReactiveCocoa",
+  "version": “2.5.2”,
```
Comment (soxjke) [inline ReactiveCocoa.podspec.json]: Don't use TextEdit - it has specific symbols in it's encoding

I suggest:
1. TextWrangler
2. SublimeText
3. vim

These quotes are not quotes and will fire an error on install

**Comment by soxjke** [comment]
This commit should also be in review https://github.com/techery/ReactiveCocoa/commit/881606068b111bc39749a3e30b01cf9b8fbb7009

**Comment by soxjke** [comment]
lgtm after fixes

**Comment by fuzza** [comment]
lgtm

**Pull request merged**